src/module/rules/rule-element/roll-option.ts

```diff
@@ -64,6 +64,7 @@ class RollOptionRuleElement extends RuleElementPF2e<RollOptionSchema> {
     static override defineSchema(): RollOptionSchema {
         return {
             ...super.defineSchema(),
+            scope: new fields.StringField({ required: false, nullable: false }),
```

Comment:
Can this be given an `initial` of the actions tab? And maybe also a `choices` of what is currently supported (two so far?)

Reply:
Until the Treat Wounds PR is done and merged only `actions-tab` is valid. I'll add the initial and choices.